core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -614,10 +614,12 @@ var ExplorationEditorMainTab = function() {
       ImageClickInput: 'General',
       ItemSelectionInput: 'General',
       MultipleChoiceInput: 'General',
+      NumericInput: 'General',
       TextInput: 'General',
       FractionInput: 'Math',
       GraphInput: 'Math',
       LogicProof: 'Math',
+      // eslint-disable-next-line no-dupe-keys
       NumericInput: 'Math',
```

Comment:
Hm, seems icky to have duplicate keys in a dict. Let's not do this.

Reply:
Maybe make the values a list of strings and make necessary changes where `INTERACTION_ID_TO_TAB_NAME` is used?

Reply:
Done.